src/app/components/WelcomeModal.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+class WelcomeModal extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            modal: this.props.show
```

Comment:
```suggestion
            modal: false
```
просто есть ли смысл привязывать `WelcomeModal` к передаваемому родителем props'у под названием `show`?:) Легче сразу в самом компоненте `WelcomeModal` уточнить значение